es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,54 @@
+import createElement from '../lib/create-element';
+import noop from '../lib/noop';
```

Comment:
While this works you should take advantage of the `index.js` file created in the `lib` folder. This allows you to import both things in 1 line. For example:

```JAVASCRIPT
import { createElement, noop } from '../lib';
```